Sparkle/SUAppcastItem.m

```diff
@@ -169,6 +175,19 @@ - (BOOL)isMacOsUpdate
     return self.osString == nil || [self.osString isEqualToString:SUAppcastAttributeValueMacOS];
 }
 
+- (NSDate *)date
+{
+    if (self.dateString == nil) {
+        return nil;
+    }
+    
+    NSDateFormatter* dateFormatter = [[NSDateFormatter alloc] init];
+    dateFormatter.locale = [[NSLocale alloc] initWithLocaleIdentifier:@"en_US"];
+    dateFormatter.dateFormat = @"E, dd MMM yyyy HH:mm:ss Z";
```

Comment:
generate_appcast uses..
`EEE, dd MMM yyyy HH:mm:ss ZZ`

In my testing, I have been using EEE (eg `Mon`) for pubDate in the test app cast, so in practice it might not matter.